base_product_mass_addition/models/product_product.py

```diff
@@ -13,25 +13,59 @@ class ProductProduct(models.Model):
 
     qty_to_process = fields.Float(
         compute="_compute_process_qty",
-        inverse="_inverse_set_process_qty",
+        inverse="_inverse_quick_vals",
         help="Set this quantity to create a new line "
         "for this product or update the existing one.",
     )
+    quick_uom_category_id = fields.Many2one(
```

Comment:
quick_uom_category_id should be a related field of quick_uom_id not need for specific compute.

fields.Many2one(related="quick_uom_id.category_id")

Indeed modules will already implement a logic for getting the default or existing quick_uom_id so we can just reuse all this logic.

Reply:
done